### PR TITLE
Update TLD prices 2025

### PR DESCRIPTION
**Pull request by ddelwig**

### CODE REVIEW

File: docs/hypernode-platform/varnish/how-to-read-varnish-stats.md
Comment (poespas) [inline docs/hypernode-platform/varnish/how-to-read-varnish-stats.md:1]: This file seems to be unrelated to the PR, please remove this :)

File: hypernode-docs-next
Comment (poespas) [inline hypernode-docs-next:1]: This also seems to be an accident push

**Review by tdgroot — APPROVED** [review]

**Review by martijneichelsheim — APPROVED** [review]
